src/middleware/validateApiKey.js

```diff
@@ -64,28 +20,20 @@ export default async (req, res, next) => {
     if (!apiKey && isDevelopment) {
       apiKey = FALLBACK_API_KEY;
     }
+
     if (!apiKey) {
-      throw new Error('X-API-Key Header doesn\'t exist');
```

Comment:
this is going to fail linting - we don't use double quotes - you really don't need to make a change to this part of the code 😅